docs/source/contributors/index.md

```diff
@@ -35,7 +35,7 @@ Due to a compatibility issue with Webpack, Node.js 18.15.0 does not work with Ju
 After you have installed the prerequisites, create a new conda environment and activate it.
 
 ```
-conda create -n jupyter-ai python=3.11 nodejs=20
+conda create -n jupyter-ai python=3.11 conda-forge::nodejs=20
```

Comment:
Usually we pass the flag `-c conda-forge` to Conda commands to indicate that users must be pulling from the `conda-forge` channel.

Can you try replacing this with the below command? This syntax is equivalent but more common in Jupyter documentation.

```
conda create -n jupyter-ai -c conda-forge python=3.11 nodejs=20
```

Reply:
This is indeed potentially a better way because it will mean both python and nodejs will be installed from single source (conda-forge) which reduces a chance of conflicts in binaries of shared libraries and such.

Reply:
@krassowski I think Python will only be pulled from `conda-forge` if `--override-channels` flag is provided as well, given a default Conda config. Here's what [the docs](https://github.com/jupyterlab/jupyter-ai/issues/680) say:

<img width="744" alt="Screenshot 2024-03-06 at 10 14 43 AM" src="https://github.com/jupyterlab/jupyter-ai/assets/44106031/d052a16b-da78-4dd2-aebe-e9d03c035ed9">

I don't think it's worth including `--override-channels` since I _assume_ that package versioning ensures that installing from either channel is equivalent. What do you think, should we or should we not include `--override-channels` with the `conda` commands in Jupyter documentation?

Reply:
I have changed it as suggested. With the default conda config, both Python and nodejs will be pulled from the conda-forge channel with the following command. 

```bash
conda create -n jupyter-ai -c conda-forge python=3.11 nodejs=20
```

![image](https://github.com/jupyterlab/jupyter-ai/assets/5016453/ab5d1456-5842-4172-93eb-4fa513a8ed52)

Reply:
@giswqs Awesome, thank you for checking and proving me wrong! 😁 

@krassowski As you can see above, please disregard my previous comment. I totally missed the sentence in the middle where it says the channels specified via `-c` are used first before the defaults. 🤦 I reviewed a ton of PRs lately so my reading comprehension has suffered. 😂